src/smart-components/group/groups.js

```diff
@@ -125,6 +147,8 @@ Groups.propTypes = {
   isLoading: PropTypes.bool,
   searchFilter: PropTypes.string,
   fetchGroups: PropTypes.func.isRequired,
+  selectedRows: PropTypes.array,
```

Comment:
I always prefer using `arrayOf(PropTypes.shape({...}))`. It makes the type checking more secure.